### ISSUE TITLE
Concerns about slow downloads of small files

### ISSUE THREAD

**Issue by liangddyy** [emanzione/PATCH#66]
The exported Unity project is quite large, about 1G+, which causes it to take longer to download the first time (because there are a lot of small files). Is the ability to compress the version package and then download it again considered?

**Comment by emanzione** [comment]
Yeah, it has been considered some time ago. However, it makes things a bit more complex on both Admin side and Updater side.

Recently, due to the Unity's pricing changes, I started a huge refactoring of the codebase and I am reconsidering some design choices. I could give builds compression another possibility, but ideally I would like to not break compatibility with artifacts produced by previous releases.

**Comment by liangddyy** [comment]
Very much looking forward to that feature if it is an option.
This feature should drastically improve the experience for first time installs of large apps (only triggered on first download).

Also I have a small suggestion:
For the launcher update, instead of putting it in the game (which currently requires a PreGame scenario to do this), I think you can refer to this launcher: 

assetstore.unity.com/packages/tools/utilities/game-launcher-patcher-and-updater-pro-237342

He provides an external exe to restart the launcher, which is a nice thought.
This would put the self updates for both the game and the launcher, into the launcher. A much cleaner process.
